.travis.yml

```diff
@@ -33,12 +33,17 @@ osx_xcode10_1: &osx_xcode10_1
 windows_vs2017: &windows_vs2017
   os: windows
 
-matrix:
+stages:
```

Comment:
Not sure if build stages are really the way to go. As it's not multiple stages to achieve a result. Can't we simply add a job conditionally? Also, by using a *stage* the rest of the jobs automatically go under the Test stage. If adding a job conditionally isn't possible I'd at least expect that the name of the stage is determined by us.